src/itp.jl

```diff
@@ -59,12 +59,12 @@ struct ITP{T} <: AbstractBracketingAlgorithm
 end
 
 function SciMLBase.solve(prob::IntervalNonlinearProblem, alg::ITP,
-    args...; abstol = 1.0e-15,
+    args...; abstol = nothing,
     maxiters = 1000, kwargs...)
     f = Base.Fix2(prob.f, prob.p)
     left, right = prob.tspan # a and b
     fl, fr = f(left), f(right)
-    ϵ = abstol
+    ϵ = abstol !== nothing ? abstol : min(eps(left), eps(right))
```

Comment:
Why is this not just done in the keyword argument to make it cleaner?

Reply:
yes this can be done to make the code cleaner. made the changes